cynthion/python/Makefile

```diff
@@ -24,13 +28,31 @@ clean:
 console:
 	pyserial-miniterm $(UART) 115200
 
-load:
-	apollo configure build/facedancer.bit
-
-tests:
+tests: .PHONY
 	python3 -m unittest
 
-.PHONY: tests
+.PHONY:
+
+
+# - development ---------------------------------------------------------------
+
+assets: assets-bitstreams assets/moondancer.bin
+
+assets-bitstreams: detect-fpga-toolchain detect-luna-platform
+	@mkdir -p 'assets/$(PLATFORM_NAME)'
+	@if [ ! -s "assets/$(PLATFORM_NAME)/analyzer.bit" ]; then \
+		python -m cynthion.gateware.analyzer.top --output 'assets/$(PLATFORM_NAME)/analyzer.bit' ; \
+	fi
+	@if [ ! -s "assets/$(PLATFORM_NAME)/facedancer.bit" ]; then \
+		python -m cynthion.gateware.facedancer.top --output 'assets/$(PLATFORM_NAME)/facedancer.bit' ; \
+	fi
+	@if [ ! -s "assets/$(PLATFORM_NAME)/selftest.bit" ]; then \
+		python -m cynthion.gateware.selftest.top --output 'assets/$(PLATFORM_NAME)/selftest.bit' ; \
+	fi
+
+assets/moondancer.bin: detect-rust-toolchain
+	cd ../../firmware/moondancer && \
+	cargo objcopy --release -- -Obinary ../../cynthion/python/assets/moondancer.bin
```

Comment:
Can we please remove `assets/` contents in `clean`?

Reply:
Also I think we should do `cargo clean` in `clean`.

Reply:
> Can we please remove `assets/` contents in `clean`?

I've added a `make distclean` that will remove all generated files in `assets/` so one can still use `make clean` for common operations without blowing everything away.

> Also I think we should do cargo clean in clean.

I'd rather not, that would add a dependency on the rust toolchain to it which would be somewhat of a pain if you're only working with gateware.

Reply:
https://github.com/greatscottgadgets/cynthion/pull/166/commits/abf58204dd405630eff918f540221ed68536411f